coopr-standalone/README.md

```diff
@@ -12,13 +12,12 @@ mvn clean package assembly:single -DskipTests
 ```
 
 ## Install
+
+This will install the appropriate Ruby Gems on the system.
 ```
-sudo gem install fog --version 1.26.0
-sudo gem install sinatra --version 1.4.5
-sudo gem install thin --version 1.6.2
-sudo gem install rest_client --version 1.7.3
-sudo gem install google-api-client --version 0.7.1
-sudo gem install deep_merge --version 1.0.1
+sudo gem install bundler
+cd coopr-provisioner
```

Comment:
I think this should match the "within standalone" context of the next section. otherwise people will be trying to startup standalone from a repo checkout.  I suggest ``bundle install --gemfile provisioner/Gemfile`` so that the ``cd`` doesn't interfere with the next section's ``bin/coopr.sh``